turbine/src/broadcast_stage/broadcast_duplicates_run.rs

```diff
@@ -207,6 +217,9 @@ impl BroadcastRun for BroadcastDuplicatesRun {
                     bank.slot(),
                     sigs,
                 );
+                if let Some(duplicate_slot_sender) = &self.config.duplicate_slot_sender {
```

Comment:
Can this be moved outside `last_entries.map`

Reply:
I don't think so since we only want to send slots that have duplicate shreds, and the signal that duplicate shreds will be sent is if  `last_entries` is Some

Reply:
I think it would be more clear if it was next to where we are sending the duplicate shreds
```
socket_sender.send((original_last_data_shred, None))?;
socket_sender.send((partition_last_data_shred, None))?;
```

Reply:
makes sense, moved